WearBuildConfig/mobile/build.gradle

```diff
@@ -0,0 +1,71 @@
+apply plugin: 'com.android.application'
+apply from: "$rootProject.projectDir/WearBuildConfig/shared-config/android-signing.gradle"
+
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.0"
+
+    defaultConfig {
+        applicationId "com.novoda.demo.wearbuildconfig"
+        minSdkVersion 16
+        targetSdkVersion 25
+        versionCode 1
+        versionName "1.0"
+
+        buildConfigField 'boolean', 'PLAY_SERVICES', "true"
```

Comment:
if you're seeking a good example of a `build.gradle` you could consider the use of the [`gradle-build-properties-plugin`](https://github.com/novoda/gradle-build-properties-plugin) and its [typed decorators for `buildConfigField`](https://github.com/novoda/gradle-build-properties-plugin#4-typed-buildconfigfield--resvalue)